cmd/mirror-main.go

```diff
@@ -542,12 +554,23 @@ func copyBucketPolicies(srcClt, dstClt Client, isForce bool) *probe.Error {
 // runMirror - mirrors all buckets to another S3 server
 func runMirror(srcURL, dstURL string, ctx *cli.Context) *probe.Error {
 
+	isFake := ctx.Bool("fake")
+	isRemove := ctx.Bool("remove")
+	isWatch := ctx.Bool("watch")
+	isForce := ctx.Bool("force")
+
+	olderThan := ctx.Int("older-than")
```

Comment:
Combine this code everywhere.. i see that parsing older-than and newer-than can be put into a single function.